tests/test_autonomy/test_cli/base.py

```diff
@@ -41,15 +41,15 @@ class BaseCliTest:
     cli_options: Tuple[str, ...]
 
     @classmethod
```

Comment:
The `setup` and `teardown` methods are called before each `test_` method. The decorator makes no sense.